spec/components/works/detail_component_spec.rb

```diff
@@ -74,6 +74,15 @@
     end
   end
 
+  describe 'contributors' do
+    let(:work_version) { build_stubbed(:work_version, authors: [author]) }
+    let(:author) { build_stubbed(:person_author, orcid: 'https://orcid.org/0000-0002-1825-0097') }
+
+    it 'renders the contributor' do
+      expect(rendered.css('#contributors a').to_html).to include('https://orcid.org/0000-0002-1825-0097')
+    end
+  end
```

Comment:
would it be more complete to have also a spec for the contributor role?   And also to have a spec for a contributor without an ORCID?  And full name vs. first name and last name?  Asking for a friend.

Reply:
I'll add one for a contributor; I wasn't looking to write complete tests for the contributors section.